modules/programs/beets.nix

```diff
@@ -48,13 +68,46 @@ in {
           <filename>$XDG_CONFIG_HOME/beets/config.yaml</filename>
         '';
       };
+
+      mpdIntegration = mkOption {
+        type = mpdIntegrationOpts;
```

Comment:
There's no reason to have a submodule here, is there? (As opposed to `mpdIntegration = { enableStats = ...; }`)

If not, I'd rather remove the submodule because of https://github.com/NixOS/nixpkgs/issues/96006.